src/dotnet/commands/dotnet-publish/Program.cs

```diff
@@ -97,7 +97,7 @@ public static PublishCommand FromArgs(string[] args, string msbuildPath = null)
 
                 if (!string.IsNullOrEmpty(filterProjOption.Value()))
                 {
-                    msbuildArgs.Add($"/p:FilterProjFile={filterProjOption.Value()}");
+                    msbuildArgs.Add($"/p:FilterProjectFiles={filterProjOption.Value()}");
```

Comment:
Yes, please add it in a subsequent PR.  Let's get the new SDK in now with minimal changes.